bears/latex/LatexLintBear.py

```diff
@@ -22,6 +22,6 @@ class LatexLintBear:
     def create_arguments(filename, file, config_file):
         return (
             '--format',
-            '%k %n in {0} line %l: %m!n'.format(filename),
+            '%k %n in {0} line %l: %m'.format(filename),
```

Comment:
hmm.  the `!n` was added by me while fixing https://github.com/coala/coala-bears/issues/317

It translates to newline, but I am not 100% sure it was necessary.

Reply:
Ya, `!n` isnt working.

Another approach is to use `-v0`